src/icons/Maximize/index.js

```diff
@@ -0,0 +1 @@
+export { default } from './Maximize';
```

Comment:
Rename components from Expand to Maximize inside Maximize.js

Reply:
@shridhar52war you did it on `minimize` but forgot in `maximize` 😆

Reply:
🤦